swift/apple/FirezoneNetworkExtension/main.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  main.swift
+//  FirezoneNetworkExtension
+//
+//  Created by Jamil Bou Kheir on 11/14/24.
+//
```

Comment:
Do we need these headers? Seems pretty redundant with what we know from Git.

Reply:
They're pretty standard in Swift / Xcode (Android too). However they weren't following our convention, so I updated them to do so.